screens/ReportsScreen.js

```diff
@@ -37,7 +78,7 @@ class ReportsScreen extends React.Component {
 
           <View style={[styles.flex(1)]}>
             <View style={[styles.menuContainer]}>
-              <MenuButton
+              {!!userScope ? <MenuButton
```

Comment:
I think my original idea is to use the component like this:

<PermissionCheckTouchable
    requiredPermission="OWNER"
    uiComponent={<MenuButton
                route='StaffTimeCard'
                onPress={() => NavigationService?.navigateToRoute('StaffTimeCard')}
                title={t('staffTimeCardReport')}
	@@ -59,11 +122,32 @@ class ReportsScreen extends React.Component {
                    size={40}
                    style={[styles?.buttonIconStyle(customMainThemeColor)]}
                  />
                }
              />}
/>